NSS-Day-Cohort-53/tabloid-full-stack-always-puffy-impalas
# View Post by Category

#### Changes Made
1. Added file `PostByCategoryList.js` to `components` folder within `client` directory.
2. Modified file `postManager.js` to include `getPostByCategoryId` 
3. Modified file `CategoryList.js` to include `Link` to posts by category.
4. Modified file `ApplicationViews.js` to include `Route` to PostByCategoryList
5. Modified file `PostRepository.cs` to include `GetPostByCategoryId` 
6. Modified `PostController.cs` to include GET method for PostByCategoryId

​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout SH-ViewPostsByCategory
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Test app functionality.
    > When a user is viewing the categories list, they can click the name of a category and see all posts within that category.
7. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

closes issue #4
